resources/education.md

```diff
@@ -0,0 +1,46 @@
+---
+parent: Resources
+title: Education
+description: Machine translation learning resources
+---
+
+
+## Machine Translation massive open online course
+
+The **Machine Translation** course is a massive open online course from Coursera.
+This course is taught by Professor Doctor Alexander Waibel and Assistant Professor Doctor Jan Niehus from Karlsruhe Institute of Technology.
+This course is designed for learners with intermediate knowledge.
```

Comment:
This doesn't really match the others, too generic.  Intermediate knowledge of computer science, I assume?